accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/AcHelper.java

```diff
@@ -58,17 +62,32 @@ private AcHelper() {
     public static AceBean getAceBean(final AceWrapper ace)
             throws ValueFormatException, IllegalStateException,
             RepositoryException {
-        AceBean aceBean = new AceBean();
+        final AceBean aceBean = new AceBean();

```

Comment:
We don't use "final as default" in this project - this was probably added by a safe action?